pallets/parachain-staking/src/set.rs

```diff
@@ -87,3 +88,84 @@ impl<T: Ord> From<Vec<T>> for OrderedSet<T> {
         Self::from(v)
     }
 }
+
+/// An ordered set backed by `BoundedVec`
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
```

Comment:
nice work with this data structure. I think it would be nice to have some unit tests for its interfaces.